content/docs/1.7.0/deploy/important-notes/index.md

```diff
@@ -8,73 +8,31 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
 
 - [General](#general)
   - [Supported Kubernetes Versions](#supported-kubernetes-versions)
-  - [Offline Upgrade Required To Fully Prevent Unexpected Replica Expansion](#offline-upgrade-required-to-fully-prevent-unexpected-replica-expansion)
-  - [Default Priority Class](#default-priority-class)
-  - [New Node Drain Policies Added](#new-node-drain-policies-added)
-  - [Custom Resource Fields Deprecated](#custom-resource-fields-deprecated)
   - [Pod Security Policies Disabled \& Pod Security Admission Introduction](#pod-security-policies-disabled--pod-security-admission-introduction)
-  - [Updating CSI Snapshot CRD `v1beta1` to `v1`, `v1beta1` Removed](#updating-csi-snapshot-crd-v1beta1-to-v1-v1beta1-removed)
-  - [Engine Upgrade Enforcement](#engine-upgrade-enforcement)
-  - [Danger Zone Setting Configuration](#danger-zone-setting-configuration)
-  - [Longhorn PVC with Block Volume Mode](#longhorn-pvc-with-block-volume-mode)
-  - [Minimum XFS Filesystem Size](#minimum-xfs-filesystem-size)
+  - [Introduction of Commandline Tool](#introduction-of-commandline-tool)
+- [Networking](#networking)
+  - [Storage Network Support for Read-Write-Many (RWX) Volumes](#storage-network-support-for-read-write-many-rwx-volumes)
+- [Resilience](#resilience)
+  - [RWX Volumes Fast Failover](#rwx-volumes-fast-failover)
+  - [Support Configurable Timeout for Replica Rebuilding and Snapshot cloning](#support-configurable-timeout-for-replica-rebuilding-and-snapshot-cloning)
+- [Data Integrity and Reliability](#data-integrity-and-reliability)
+  - [Support Periodic and On-Demand Full Backups to Enhance Backup Reliability](#support-periodic-and-on-demand-full-backups-to-enhance-backup-reliability)
+  - [High Availability of Backing Images](#high-availability-of-backing-images)
 - [V2 Data Engine](#v2-data-engine)
   - [Longhorn System Upgrade](#longhorn-system-upgrade)
```

Comment:
Better _Longhorn System Live Upgrade_?

Reply:
No, it is not related to live upgrade. Before introducing the control plane of live upgrade, users need to detach volumes.